obstacle_range: 6.0
raytrace_range: 8.5
footprint: [[0.0012, 0.0014], [0.0012, -0.0014], [-0.0012, -0.0014], [-0.0012, 0.0014]]
footprint_padding: 0.003 #
inflation_radius: 0.55 #
transform_tolerance: 1 #
controller_patience: 2.0 #
NavfnROS: #
    allow_unknown: true #
recovery_behaviors: [ #
    {name: conservative_clear, type: clear_costmap_recovery/ClearCostmapRecovery}, #
    {name: aggressive_clear, type: clear_costmap_recovery/ClearCostmapRecovery} #
] #

conservative_clear: #
    reset_distance: 3.00 #
aggressive_clear: #
    reset_distance: 1.84 #
#map_topic: /map
subscribe_to_updates: true
# observation_sources: point_cloud_sensor
# point_cloud_sensor: {sensor_frame: map, data_type: PointCloud, topic: /rtabmap/mapData, marking: true, clearing: true}
global_frame: map
robot_base_frame: base_link
always_send_full_costmap: true